File: rss-reader/app/src/main/java/com/abubusoft/kripton/examples/rssreader/RssApplication.java
package com.abubusoft.kripton.examples.rssreader;

import android.app.Application;

import com.abubusoft.kripton.android.KriptonLibrary;

public class RssApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        KriptonLibrary.init(this);
    }
}

File: kotlin-rss-reader/app/src/main/java/com/abubusoft/kripton/example/rssreader/RssApplication.kt
package com.abubusoft.kripton.example.rssreader

import com.abubusoft.kripton.android.KriptonLibrary
import android.app.Application


class RssApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        KriptonLibrary.init(this)
    }
}
